src/orbitals.jl

```diff
@@ -60,6 +67,13 @@ struct Orbital{N<:MQ} <: AbstractOrbital
     end
 end
 
+"""
+    mqtype(::Orbital{MQ}) = MQ
```

Comment:
I called it main quantum number in `orbitals.jl`, because I did not know any better :) but you've (correctly) called it _principal quantum number_ everywhere else, so maybe here as well to be consistent?

Reply:
E.g. `prinqtype`, maybe `pqtype` is a bit too short and nondescript.

Reply:
I called it `mqtype` because we use `MQ` as the name of the type variable. As this is just an internal method, I think it is best if it is consistent with that.

We could change `MQ` everywhere to `PQ` or `PQN`. I guess it would be slightly better so that it would be consistent with the term itself, but I'd do that in a separate PR then. Regarding the terminology itself, I hadn't really thought about which term would be best to use, so I defer to [Wikipedia](https://en.wikipedia.org/wiki/Principal_quantum_number) as the definitive source.